pkg/osquery/table/airdrop_darwin.go

```diff
@@ -54,7 +54,27 @@ func (t *airdropTable) generateAirdrop(ctx context.Context, queryContext table.Q
 }
 
 func queryPrimaryUser(client *osquery.ExtensionManagerClient) (string, error) {
```

Comment:
After much discussion, we refactored the recent table additions to return results for all users rather than using this (expensive) primary user query. Looks like I missed this one.

See for example https://github.com/kolide/launcher/blob/master/pkg/osquery/table/gdrive_sync.go#L95-L102.

What if we extract this to a `primary_console_user` table?